### ISSUE TITLE
I can not find the code for different normalization?

### ISSUE THREAD

**Issue by hiyyg** [XiaoYunZhou27/UNet-with-BN-GN-IN-LN#1]
Hi @XiaoYunZhou27 , very interesting work! Could you please point me out where is the code for different normalization?

**Comment by nzw0301** [comment]
Me neither. I suppose all `dropout` after `conv2d`/`conv2d_transpose` and before `relu` should be replaced with normalisation layers.

**Comment by adolfogc** [comment]
Hi @hiyyg, @nzw0301, I was wondering the same while looking at the code, but page 4 of the paper answered this question for me:

> The programming of BN is from tf.layers while that of IN, LN and GN are from tf.contrib.layers.
